src/style.css

```diff
@@ -0,0 +1,108 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap");
+
+.page_content {
+  margin: 0;
+  background-color: #444;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card_img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+}
+
+.card_product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 40px 16px 0;
+}
+
+.card_product-code {
+  margin-top: 4px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-left: 16px;
+}
+
+.card_viewer-info-block {
+  margin: 16px 16px 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+.star {
+  display: flex;
+}
+
+.star_item {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url(./images/star-active.svg) center no-repeat;
+}
+
+.star_item:last-child {
+  background: url(./images/star.svg) center no-repeat;
+}
```

Comment:
p. 11 of checklist [FILE STRUCTURE] - You should create separate file for each BEM block